tests/functional/test_thermal_heuristic_two_clusters_low_load.py

```diff
@@ -0,0 +1,269 @@
+        idx_generation=4, idx_nodu=8, idx_spillage=11, idx_unsupplied=10
+    )
+
+    thermal_problem_builder = ThermalProblemBuilder(
+        fast=False,
+        data_dir=Path(__file__).parent / data_path,
+        id_thermal_cluster_model=THERMAL_CLUSTER_MODEL_MILP.id,
+        port_types=[BALANCE_PORT_TYPE],
+        models=[AccurateModelBuilder(THERMAL_CLUSTER_MODEL_MILP).model] + models,
+        time_scenario_hour_parameter=TimeScenarioHourParameter(1, 1, 168),
+    )
+
+    for scenario in range(
+        thermal_problem_builder.time_scenario_hour_parameter.scenario
+    ):
+        for week in range(thermal_problem_builder.time_scenario_hour_parameter.week):
+            week_scenario_index = WeekScenarioIndex(week, scenario)
+            # First optimization
+            resolution_step_1 = thermal_problem_builder.main_resolution_step(
+                week_scenario_index,
+                solving_parameters=SolvingParameters(solver_parameters),
+            )
+
+            thermal_problem_builder.update_database_heuristic(
+                resolution_step_1.output,
+                week_scenario_index,
+                None,
+                param_to_update="nb_units_min",
+                var_to_read="nb_on",
+                fn_to_apply=lambda x: ceil(round(x, 12)),
+            )
+
+            for g in thermal_problem_builder.heuristic_components():
+                # Solve heuristic problem
+                resolution_step_accurate_heuristic = (
+                    thermal_problem_builder.heuristic_resolution_step(
+                        week_scenario_index,
+                        id_component=g,
+                        model=HeuristicAccurateModelBuilder(
+                            THERMAL_CLUSTER_MODEL_MILP
+                        ).model,
+                        solving_parameters=SolvingParameters(solver_parameters),
+                    )
+                )
+
+                thermal_problem_builder.update_database_heuristic(
+                    resolution_step_accurate_heuristic.output,
+                    week_scenario_index,
+                    [g],
+                    param_to_update="nb_units_min",
+                    var_to_read="nb_on",
+                    fn_to_apply=lambda x: ceil(round(x, 12)),
+                )
+
+            # Second optimization with lower bound modified
+            resolution_step_2 = thermal_problem_builder.main_resolution_step(
+                week_scenario_index,
+                solving_parameters=SolvingParameters(solver_parameters),
+            )
+
+            expected_output = ExpectedOutput(
+                mode="accurate",
+                index=week_scenario_index,
+                dir_path=data_path,
+                list_cluster=thermal_problem_builder.heuristic_components(),
+                output_idx=output_indexes,
+            )
+            expected_output.check_output_values(resolution_step_2.output)
+
+            expected_cost = [[36060641]]
```

Comment:
You perform checks only on the cost, but you mention spillage in the description, is it possible to check the spillage too ?